R/create_title.R

```diff
@@ -21,6 +21,9 @@ create_title <- function(
     year <- format(Sys.Date(), "%Y")
   }
 
+  # Species latin name with italics latex fxn
+  spp_latin <- paste("\\textit{", spp_latin, "}", sep = "")
```

Comment:
The final string, in latex, should have one backslash. Does adding two here have a purpose for, e.g., escaping recognition in R?

Reply:
Yes, its necessary. When using cat() to form the tex file, the double slash is needed otherwise it messes up the formatting :/